app/src/main/java/org/gnucash/android/ui/account/AccountsListFragment.java

```diff
@@ -265,10 +269,11 @@ public void tryDeleteAccount(long rowId) {
      * @param id Record ID of account to be deleted after confirmation
      */
     public void showConfirmationDialog(long id) {
```

Comment:
Should this be:
```
public void showConfirmationDialog(long id) {
```
like here in this PR
or:
```
private void showConfirmationDialog(String accountUID) {
```
like in your previous commit https://github.com/GnuCash-Pocket/gnucash-android/commit/b698dc4a809c6589beb69e193f7c9252947773c2 ?

Reply:
`private` is ok